### PR TITLE
Improve Kubernetes configuration management

### PR DESCRIPTION
**Pull request by willthames**
##### SUMMARY
Install openshift python module to allow use of k8s module for managing kubernetes

Also update botocore to very latest version to allow EKS management

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - Installer

##### AWX VERSION

```
awx: 1.0.6.16
```

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -218,12 +229,13 @@ twilio==6.10.4
...
 uwsgitop==0.10.0
-websocket-client==0.47.0  # via slackclient
+websocket-client==0.47.0  # via kubernetes, slackclient
 xmltodict==0.11.0
 zope.interface==4.4.3     # via twisted
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==9.0.1
-setuptools==36.0.1
+# pip
```
Comment (willthames) [inline requirements/requirements.txt]: I don't know if this change is expected or not, this file is as output by the pip-compile instructions at the top of the file.

**Comment by matburt** [comment]
Some of these changes are going to have a direct impact on other components such as job running and syncing inventory from ec2. What sort of testing have you done to verify that these dependency changes don't cause other issues?

**Comment by willthames** [comment]
@matburt in general all botocore changes are backward compatible. I'm on the Ansible AWS working group so I have a pretty good understanding of the impact of boto3 and botocore updates (there's almost none) 

However, I haven't thoroughly tested this so if you want me to revert the boto3/botocore changes, I can.

**Comment by willthames** [comment]
Let me know if you have concerns over other changes - they're all working for us but I am a little concerned about the pip and setuptools change at the bottom.

**Comment by shanemcd** [comment]
Unfortunately enough time has passed where there are now conflicts in this PR and I realize it's been almost 3 months since anyone has gotten back to you.

With the amount of testing required in order to change or modify our dependencies, it's hard for us to receive a contribution like this and also line up with our bandwidth to test and support it.

If there is a technical reason for needing the newer version of boto3, I think the best course of action would be to open a new issue so that it can be triaged and fit into a future release.

**Pull request closed without merging**